pkg/utils/opsqueue.go

```diff
@@ -0,0 +1,55 @@
+package utils
+
+import (
+	"sync"
+
+	"github.com/livekit/protocol/utils"
+)
+
+const (
+	MaxOps = 50
+)
+
+type OpsQueue struct {
+	lock      sync.RWMutex
+	ops       chan func()
+	isStopped utils.AtomicFlag
```

Comment:
we have a lock here, can use bool value

Reply:
good point, will change. I like the `TrySet`, so didn't think much :-)